### ISSUE TITLE
editor.html.get() returns target even if it's not allowed

### ISSUE THREAD

**Issue by xmedeko** [froala/wysiwyg-editor#3707]
##### Steps to reproduce the problem.
I have **not** `target` in `htmlAllowedAttrs` and I have `linkAlwaysBlank=true` to remove the target checkbox for links. When I do `editor.html.get()`, it returns link `<a href="..." target="_blank">...</a>`. However, when I do just
```js
var html = editor.html.get()
editor.html.set(html)
html = editor.html.get()
```
without editing, then the `target` attribute is removed. That's inconsistent behaviour.

##### Expected behavior.
Do not return `target` when it's not allowed in `htmlAllowedAttrs`.

##### Editor version.
v3.0.5

##### OS.
Windows 10

##### Browser.
Electron  v5.0.11

**Comment by casualuser** [comment]
Hello,
We've tried to reproduce the issue with last master branch code (3.0.6 and pre-3.1) but it looks like it works fine here - https://jsfiddle.net/froala/9ew0vqdj/
Please check this at your side and let us know if it still persists in your environment then share a repro.

**Comment by xmedeko** [comment]
@casualuser The problem is, Froala do `set()` on init in your fiddle. Try this https://jsfiddle.net/7pv20snq/ - **first edit the link**, then press "Froala get" button. (Editing is necessary to simulate this bug).

**Comment by casualuser** [comment]
@xmedeko
so it looks question in allowed attr list 
try this - https://jsfiddle.net/froala/k314r25d/
if `target` and `rel` added then everything works as expected
it looks like in this case `linkAlwaysBlank` depends on `htmlAllowedAttrs`

**Comment by xmedeko** [comment]
Yes, if you add `target, rel` attributes, it works as expected. I reports this bug as:
1. Inconistent behaviour. Froala behaviour should be consistent and I would expect same result from `get()` and does not matter if the `set()` is called or not. That;s the clear bug.
2. The problem between missing `rel, target` in  `htmlAllowedAttrs` and the link with target. Note, the problem is not in the `linkAlwaysBlank` only, the problem is event  when the user manually check checkbox "Open in a new tab". You should define behaviour in such case, document it and make it consistent for all use cases (see point 1.).

**Comment by ilyaskarim** [comment]
We’re closing this issue due to inactivity. If you’re still experiencing this problem on the latest version of our editor, please feel free to reopen the issue and let us know. You can find details on the latest updates here: [Froala Editor Changelog](https://froala.com/wysiwyg-editor/changelog/).

**Comment by xmedeko** [comment]
@ilyaskarim Problem is still in Froala 4.4.0, I have updated the first post with jsfiddle how to reproduce the bug in the current Froala version. Please, reopen the issue, I cannot do it.
